woocommerce/google-listings-and-ads
# Shows a Jetpack connected success text when viewing MC setup and settings pages as a non jetpack owner

### Changes proposed in this Pull Request:

Closes #624 

- Add a util function to get the connected jetpack info
- Show jetpack connected info according to whether the current user is the jetpack owner on the MC setup and Setting pages

### Screenshots:

#### MC setup

![image](https://user-images.githubusercontent.com/17420811/126432895-1d485372-bd5a-4892-bf0f-b25d59a7f4f3.png)

#### Settings

![image](https://user-images.githubusercontent.com/17420811/126433107-a1265cbe-924d-4442-bb17-ae4f4dca25a5.png)

### Detailed test instructions:

1. Log in as the jetpack owner user
2. Go to the MC setup and settings pages to see if it shows the owner email
3. Create a new admin user and log in as that user
4. Go to the MC setup and settings pages to see if it shows "Successfully connected through Jetpack"

### Changelog entry

> Fix - Shows a Jetpack connected success text instead of blank when viewing the onboarding setup and the settings pages as a non jetpack owner account.
